mod/wikirate/set/all/helper.rb

```diff
@@ -5,7 +5,6 @@ def with_header header, level: 3
 
   def standard_pointer_nest codename
     field_nest codename, view: :titled,
-                         cache: :never,
```

Comment:
I could be talked out of this, but generally turning off the cache for pointers shouldn't be necessary, and it certainly shouldn't be standard.